bootstrap/cmd/kfctl/cmd/apply.go

```diff
@@ -90,8 +92,10 @@ func init() {
 	applyCmd.PersistentFlags().StringVarP(&configFilePath, string(kftypes.FILE), "f", "",
 		`Static config file to use. Can be either a local path or a URL.
 For example:
---file=https://raw.githubusercontent.com/kubeflow/kubeflow/master/bootstrap/config/kfctl_platform_existing.yaml
---file=kfctl_platform_gcp.yaml`)
+export CONFIG=https://raw.githubusercontent.com/kubeflow/kubeflow/master/bootstrap/config/kfctl_platform_existing.yaml
```

Comment:
nit: Can we define a constant for the config file to print out in the command line? I have a feeling people will end up trying whatever we put in the help so we will want to update it over time.

I think we might want to default to something on the v0.7-branch.

Reply:
done